tests/unit/legacy-0-6-x/test-module-for-component-test.js

```diff
@@ -498,11 +501,14 @@ test('it supports dom triggered focus events', function(assert) {
   let input = this._element.querySelector('input');
   assert.equal(input.value, 'init');
 
-  focus(input);
-  assert.equal(input.value, 'focusin');
-
-  blur(input);
-  assert.equal(input.value, 'focusout');
+  return focus(input)
+    .then(() => {
+      assert.equal(input.value, 'focusin');
+      return blur(input);
+    })
+    .then(() => {
+      assert.equal(input.value, 'focusout');
+    });
```

Comment:
y u no async/await?

Reply:
Haha. I’m trying to touch those “legacy” tests as little as possible. Basically they are acting as my canary in the coal mine around if I broke existing `moduleFor*` APIs.